Installation/CHANGES.md

```diff
@@ -34,7 +34,7 @@ Release date: December 2021
 
 ### [The Heat Method](https://doc.cgal.org/5.4/Manual/packages.html#PkgHeatMethod)
 
--   Added the operator `operator(const Vector_3& v)`, which computes the squared length of `v`, to the `HeatMethodTraits_3` concept.
+-   **Breaking change**: Added the operator `operator(const Vector_3& v)`, which computes the squared length of `v`, to the `HeatMethodTraits_3` concept.
```

Comment:
I do not understand that line. I think the addition to the traits class concept is the functor `Compute_squared_length_3`, that has an `operator()(const  Vector_3&)`...

Reply:
Yes, you are right. I copied the line but it should be rephrased.